src/crab/ebpf_domain.cpp

```diff
@@ -586,8 +586,7 @@ assume_unsigned_64bit_gt(const NumAbsDomain& inv, bool strict, variable_t left_s
     }
 }
 
-static std::vector<linear_constraint_t>
-assume_unsigned_32bit_gt(const NumAbsDomain& inv, bool strict, variable_t left_svalue, variable_t left_uvalue,
+static std::vector<linear_constraint_t> assume_unsigned_32bit_gt(const NumAbsDomain& inv, bool strict, variable_t left_svalue, variable_t left_uvalue,
```

Comment:
This change was done in this line but there were a bunch of other similar line breaks added in the PR too

Reply:
I see

Reply:
Okay this is just how AlwaysBreakAfterReturnType=None option works in clang-format for PenaltyReturnTypeOnItsOwnLine=60 as is default.

(I'm waiting for clang-format 17 to always enforce clang-formatting, but it should be useful to run it anyway, when there are no C++-style attributes).